pom.xml

```diff
@@ -160,6 +160,7 @@ under the License.
           <artifactId>apache-rat-plugin</artifactId>
           <configuration>
             <excludes combine.children="append">
+              <exclude>src/main/resources/images/logos/maven-feather.svg</exclude>
```

Comment:
You have added a PNG, but excluding a SVG?

Reply:
Indeed, I've redone the commit